src/app/services/startup/startup.service.ts

```diff
@@ -235,6 +289,27 @@ export class ArlasStartupService {
         });
     }
 
+    /**
+    * Lists the fields of `collectionName` that are available for exploration with `arlasExploreApi`
+    * @param collectionName collection name
+    * @returns available fields
+    */
+    public listAvailableFields(collectionName: string): Promise<Set<string>> {
+        let availableFields = new Set<string>();
```

Comment:
Add hiddenAvailable fields like we did [here](https://github.com/gisaia/ARLAS-wui-toolkit/blob/develop/src/app/services/startup/startup.service.ts#L494)